src/components/comment-list-item.component.js

```diff
@@ -146,102 +145,108 @@ class CommentListItemComponent extends Component {
       !Object.prototype.hasOwnProperty.call(comment, 'repository_url');
 
     return (
-      <View style={styles.container}>
-        <View style={styles.header}>
-          {comment.user && (
-            <TouchableOpacity
-              style={styles.avatarContainer}
-              onPress={() =>
-                navigation.navigate(
-                  authUser.login === comment.user.login
-                    ? 'AuthProfile'
-                    : 'Profile',
-                  {
-                    user: comment.user,
-                  }
-                )}
-            >
-              <Image
-                style={styles.avatar}
-                source={{
-                  uri: comment.user.avatar_url,
-                }}
-              />
-            </TouchableOpacity>
-          )}
+      <TouchableWithoutFeedback onLongPress={this.showMenu}>
```

Comment:
Maybe use `TouchableOpacity` better to show feedback?

Reply:
yeah, looks better!